scripts/sql/32603301_reference-5-1-0.down.sql

```diff
@@ -0,0 +1,5 @@
+DELETE FROM global_strategy_metadata_chart_ref_mapping WHERE chart_ref_id=(select id from chart_ref where version='5.1.0' and name is null);
+
+DELETE FROM "public"."chart_ref" WHERE ("location" = 'reference-chart_5-1-0' AND "version" = '5.1.0');
+
+UPDATE "public"."chart_ref" SET "is_default" = 't' WHERE "location" = 'deployment-chart_4-20-0' AND "version" = '4.20.0';
```

Comment:
This need to be change from **deployment-chart_4-20-0** to **reference-chart_4-20-0**